components/clp-mcp-server/clp_mcp_server/server/server.py

```
@@ -0,0 +1,80 @@
+    :raise: Propagates `FastMCP.__init__`'s exceptions.
+    :raise: Propagates `FastMCP.tool`'s exceptions.
+    """
+    mcp = FastMCP(name=ProtocolConstant.SERVER_NAME)
+
+    @mcp.custom_route("/health", methods=["GET"])
+    async def health_check(_request: Request) -> JSONResponse:
+        """Health check endpoint.
+        
+        :arg _request: The incoming request object.
+        :return: A JSON response indicating the health status of the server.
+        """
```

Ruff (0.13.3)

Multi-line docstring summary should start at the second line

Insert line break and indentation after opening quotes

(D213)